supertokens_python/recipe/emailverification/recipe.py

```diff
@@ -341,7 +340,14 @@ async def email_verify_post(
         )
         if isinstance(response, VerifyEmailUsingTokenOkResult):
             if session is not None:
-                await session.fetch_and_set_claim(EmailVerificationClaim, user_context)
+                try:
+                    await session.fetch_and_set_claim(
+                        EmailVerificationClaim, user_context
+                    )
+                except Exception as e:
+                    # This should never happen since we have just set thestatus above
+                    if isinstance(e, UnknownUserIdError):
```

Comment:
Change to check that the exception message to check if the message is `UNKNOWN_USER_ID`

Reply:
If it's not that, you need to rethrow `e`

Reply:
> If it's not that, you need to rethrow e

I did this. But I don't think this is happening with the Node SDK.